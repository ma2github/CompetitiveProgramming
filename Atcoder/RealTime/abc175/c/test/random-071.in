-220643978260975 358520161934786 711511795076352
