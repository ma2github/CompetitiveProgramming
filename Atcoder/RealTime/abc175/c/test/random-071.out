220643978260975
